py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -252,9 +263,16 @@ def _get_connection_manager(self):
 
         return urllib3.PoolManager(**pool_manager_init_args)
 
-    def __init__(self, remote_server_addr: str, keep_alive: bool = False, ignore_proxy: bool = False):
+    def __init__(
+        self,
+        remote_server_addr: str,
+        keep_alive: bool = False,
+        ignore_proxy: bool = False,
+        ignore_certificates: bool = False,
```

Comment:
[nits] Appium Python client use `strict_ssl` here as it refers to webdriverio

Reply:
@KazuCocoa what should I do here, can you explain a bit more on what needs to be updated as per Appium's requirement?

Reply:
I'm not sure if selenium has the same argument in other bindings.

This is in Appium project's one by following webdriverio's term. We're fin to go with this, then Appium Python client will internally convert the `strict_ssl` to `ignore_certificates` for selenium bindings.

Reply:
Let's go with the current implementation